packages/clay-management-toolbar/src/ActiveState.tsx

```diff
@@ -0,0 +1,252 @@
+/**
+ * © 2019 Liferay, Inc. <https://liferay.com>
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+import ClayButton from '@clayui/button';
+import ClayIcon from '@clayui/icon';
+import ClayLink from '@clayui/link';
+import Item from './Item';
+import ItemList from './ItemList';
+import React from 'react';
+import {ClayCheckbox} from '@clayui/form';
+import {ClayDropDownWithBasicItems} from '@clayui/drop-down';
+import {IItem} from '@clayui/drop-down';
+import {IPropsBase} from './types';
+import {sub} from '@clayui/shared';
+
+interface IActionItems extends IItem {
+	quickAction?: boolean;
+}
+
+interface IProps extends IPropsBase {
+	actionItems?: Array<IActionItems>;
+	clearSelectionURL?: string;
+	onClearSelectionClick?: (
+		e: React.MouseEvent<HTMLButtonElement, MouseEvent>
+	) => void;
+	onQuickActionClick?: (
+		e: React.MouseEvent<HTMLButtonElement, MouseEvent>
+	) => void;
+	onSelectAllClick?: (
+		e: React.MouseEvent<HTMLButtonElement, MouseEvent>
+	) => void;
+	selectAllURL?: string;
+	selectedItems?: number;
+	showSelectAllButton?: boolean;
+	supportsBulkActions?: boolean;
+	totalItems?: number;
+}
+
+const ActiveState: React.FunctionComponent<IProps> = ({
+	actionItems = [],
+	checkboxChecked = false,
+	clearSelectionURL,
+	disabled = false,
+	filterItems,
+	labels = {
+		allSelected: 'All Selected',
+		checkbox: 'Select Items',
+		clearSelection: 'Clear',
+		selectAll: 'Select all',
+		selected: 'Selected',
+		selectedItemsOfTotalItems: '{0} of {1}',
+	},
+	onCheckboxChange = () => {},
```

Comment:
I'm pretty sure this needs to not be a no-op. Otherwise, if a user doesn't provide this prop, the component will show the checkbox, but won't function.